src/pages/_index/_components/what-is-wasmcloud/index.tsx

```diff
@@ -0,0 +1,194 @@
+import React from 'react';
+import { Section } from '@site/src/pages/_components/section';
+import { SectionTag } from '@site/src/pages/_components/section-tag';
+import { SectionHeading } from '@site/src/pages/_components/section-heading';
+import {
+  Switcher,
+  SwitcherButton,
+  SwitcherContent,
+  SwitcherList,
+} from '../what-is-wasmcloud/switcher';
+import { SectionSubheading } from '@site/src/pages/_components/section-subheading';
+import styles from './what-is-wasmcloud.module.css';
+
+type Props = {};
+
+const INTRO_CONTENT: {
+  tag: React.ReactNode;
+  heading: React.ReactNode;
+  description: React.ReactNode;
+} = {
+  tag: 'What is wasmCloud?',
+  heading: 'Fast-track the development, deployment, and management of distributed applications',
+  description: (
+    <p>
+      wasmCloud is an open source project from the Cloud Native Computing Foundation (CNCF) that
+      enables teams to build polyglot applications composed of reusable code and run
+      them—resiliently and efficiently—across any Kubernetes cluster, cloud, datacenter, or edge.
+    </p>
+  ),
+};
+
+const SWITCHER_CONTENT: Array<{
+  id: string;
+  title: React.ReactNode;
+  image: string;
+  features: Array<{
+    title: React.ReactNode;
+    description: React.ReactNode;
+    link?: string;
+    linkText?: React.ReactNode;
+  }>;
+}> = [
+  {
+    id: 'build',
+    image: '/img/pages/home/what-is-wasmcloud/build.svg',
+    title: 'Build',
+    features: [
+      {
+        title: 'Integrate with your existing ecosystem',
+        description: (
+          <>
+            wasmCloud has first-tier support for Kubernetes, VS Code, ArgoCD, Backstage, Chainguard,
```

Comment:
```suggestion
            wasmCloud has first-tier integrations for Kubernetes, VS Code, ArgoCD, Backstage, Chainguard,
```

This might be a little repetitious , but I feel it fits well.